sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -2032,11 +2128,134 @@ def hpat_pandas_series_mul_number_impl(self, other, level=None, fill_value=None,
     raise TypingError('{} The object must be a pandas.series or scalar. Given other: {}'.format(_func_name, other))
 
 
-@sdc_overload_method(SeriesType, 'div')
-@sdc_overload_method(SeriesType, 'truediv')
+@overload_method(SeriesType, 'div')
 def hpat_pandas_series_div(self, other, level=None, fill_value=None, axis=0):
     """
-    Pandas Series method :meth:`pandas.Series.div` and :meth:`pandas.Series.truediv` implementation.
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.Series.div
+
+    Examples
+    --------
+    .. literalinclude:: ../../../examples/series/series_div.py
+       :language: python
+       :lines: 27-
+       :caption: Element-wise division of one Series by another (binary operator div)
+       :name: ex_series_div
+
+    .. code-block:: console
+
+        > python ./series/series_div.py
+        0    0.25
+        1    0.50
+        2    0.25
+        3     NaN
+        dtype: float64
+
+    .. note::
+
+        Parameters level, fill_value, axis are currently unsupported by Intel Scalable Dataframe Compiler
+
+    .. seealso::
+
+        :ref:`Series.rdiv <pandas.Series.rdiv>`
+
+    Intel Scalable Dataframe Compiler Developer Guide
+    *************************************************
+    Pandas Series method :meth:`pandas.Series.div` implementation.
+    .. only:: developer
+
+       Test: python -m sdc.runtests sdc.tests.test_series.TestSeries.test_series_op5
+
+    Parameters
+    ----------
+    self: :class:`pandas.Series`
+        input arg
+    other: :obj:`pandas.Series`, :obj:`int` or :obj:`float`
+        input arg
+    level: :obj:`int` or name
+         *unsupported*
+    fill_value: :obj:`float` or None, default None
+              *unsupported*
+    axis: :obj:`int` default 0
+         *unsupported*
+    Returns
+    -------
+    :obj:`pandas.Series`
+         returns :obj:`pandas.Series` object
+    """
+
+    _func_name = 'Method div() or truediv().'
+
+    if not isinstance(self, SeriesType):
+        raise TypingError('{} The object must be a pandas.series. Given: {}'.format(_func_name, self))
```

Comment:
If you are changing the part of the code, let's use `TypeChecker` in this overload.